roles/StackStorm.ewc_smoketests/tasks/main.yml

```diff
@@ -1,89 +0,0 @@
----
-
-# Small suite of smoke tests to ensure that EWC role has deployed as expected
-
-- name: auth | Add a new ewc_smoke_tests_user in st2 htpasswd file
-  become: true
-  htpasswd:
-    path: /etc/st2/htpasswd
-    name: "{{ ewc_smoke_tests_user }}"
-    password: "{{ ewc_smoke_tests_password }}"
-  changed_when: no
-  notify:
-    - restart st2api
-    - restart st2stream
-  tags:
-    - ewc-smoke-tests
-
-- name: Copy smoke tests RBAC roles to /opt/stackstorm/rbac/roles directory
-  become: yes
-  template:
-    src: rbac_roles/roles.yml.j2
-    dest: /opt/stackstorm/rbac/roles/{{ item.name }}.yaml
-    owner: st2
-    group: st2
-  loop: "{{ ewc_smoke_tests_rbac.roles }}"
-  changed_when: no
-  when: ewc_smoke_tests_rbac.roles is defined
-  notify:
-    - reload ewc_rbac
-  tags:
-    - ewc-smoke-tests
-
-- name: Copy smoke tests RBAC assignments to /opt/stackstorm/rbac/assignments directory
-  become: yes
-  template:
-    src: rbac_assignments/assignments.yml.j2
-    dest: /opt/stackstorm/rbac/assignments/{{ item.name }}.yaml
-    owner: st2
-    group: st2
-  loop: "{{ ewc_smoke_tests_rbac.assignments }}"
-  changed_when: no
-  when: ewc_smoke_tests_rbac.assignments is defined
-  notify:
-    - reload ewc_rbac
-  tags:
-    - ewc-smoke-tests
-
-- meta: flush_handlers
-  tags:
-    - ewc-smoke-tests
-
-- name: Get authentication token for ewc_smoke_tests  # Note this will not use LDAP.
-  command: st2 auth "{{ ewc_smoke_tests_user }}" -p "{{ ewc_smoke_tests_password }}"  -t
-  register: st2_token_smoke_tests_user
-  changed_when: no
-  tags:
-    - ewc-smoke-tests
-
-- name: Test a simple core.local action as user ``ewc_smoke_tests_user``
-  command: st2 run core.local -- date -R
-  environment:
-    ST2_AUTH_TOKEN: "{{ st2_token_smoke_tests_user.stdout }}"
-  changed_when: no
-  tags:
-    - ewc-smoke-tests
-
-- name: Test some other action that "{{ ewc_smoke_tests_user }}" cannot run
-  command: st2 run core.http url="https://www.google.com"
-  environment:
-    ST2_AUTH_TOKEN: "{{ st2_token_smoke_tests_user.stdout }}"
-  ignore_errors: yes
-  changed_when: no
-  register: ewc_smoke_tests_forbidden_action
-  tags:
-    - ewc-smoke-tests
-
-- name: Assert forbidden error was indeed thrown
-  fail:
-    msg: "St2 action was forbidden to run because of RBAC permissions but action still ran."
-  changed_when: no
-  when: "ewc_smoke_tests_forbidden_action.stdout.find('Forbidden') == -1"
-  tags:
-    - ewc-smoke-tests
-
-- name: Teardown test artifacts
-  import_tasks: teardown.yml
```

Comment:
We want to keep the teardown logic to make sure infrastructure before and after smoke tests is in the same state.

Reply:
It's still there, I just moved the 3 tasks to the st2rbac_verification.yml: https://github.com/StackStorm/ansible-st2/blob/0ed1247c638801a4936ba2cdb20439fba113bdfe/roles/StackStorm.st2smoketests/tasks/st2rbac_verification.yml#L67

Let me know if we want to keep the tear down tasks in a dedicated file.